readme-docs/archive/readme-v3.1.2-release.md

```diff
@@ -0,0 +1,63 @@
+# Frames Android SDK
+[![CircleCI](https://circleci.com/gh/checkout/frames-android/tree/master.svg?style=svg)](https://circleci.com/gh/checkout/frames-android/tree/master)
```

Comment:
Do we use Circle CI ?

Reply:
No need to review the previous readme file in archive.
archive/readme-v3.1.2-release.md